src/owl/dense/owl_dense_matrix_generic.ml

```diff
@@ -600,6 +600,15 @@ let load_txt ?(sep="\t") k f =
   close_in h; x
 
 
+let save_npy x ~out = Npy.write x out
+
+
+let load_npy kind file =
+  match Npy.read_copy file |> Npy.to_bigarray Bigarray.c_layout kind with
+  | Some x -> x
+  | None -> failwith Printf.(sprintf "%s: incorrect format" file)
```

Comment:
A tiny arrow match issue here, but should be no problem as we are increasingly using ocamlformat :)